samples/timelapse-sample/timelapse_summary_test.cpp

```diff
@@ -0,0 +1,17 @@
+#include <boost/test/unit_test.hpp>
+#include <chrono>
+#include "timelapse_summary.h"
+
+BOOST_AUTO_TEST_SUITE(generate_timelapse)
+
+BOOST_AUTO_TEST_CASE(generateTimeLapse)
+{
+	auto timelapsePipeline = boost::shared_ptr<TimelapsePipeline>(new TimelapsePipeline());
+	timelapsePipeline -> setupPipeline();
+
+	timelapsePipeline -> startPipeline();
+
+	timelapsePipeline -> stopPipeline();
```

Comment:
this is not a test. it has no checks. also, need to use step() instead

Reply:
Done added an assertion to check the frame size.